src/modules/userSyncTest/getSettingsId.js

```diff
@@ -0,0 +1,4 @@
+export const getSettingsId = array => {
+    array[1].id = array[0]
```

Comment:
It is surprising to me that function called `getSettingsId` is mutating the argument, should this function have a different name or return a new entry without mutating the passed-in array?

Reply:
Here I'm renaming the function